gf-webapp/src/logic/rooms/roomsPage/roomBlock/RoomBlock.tsx

```diff
@@ -26,10 +29,18 @@ export const RoomsBlock = () => {
   };
 
   // if data not initialized yet
+  useEffect(() => {
+    if (!rooms) {
+      handleDataLoading();
+    }
+  }, []);
+
   if (!rooms) {
-    return <Loader />;
+    return <></>;
```

Comment:
null

Reply:
I didn't know that I can use null :)
Done